contrib/babelfishpg_tds/src/backend/tds/tdsbulkload.c

```diff
@@ -1012,7 +1024,7 @@ ReadBcpPlp(ParameterToken temp, StringInfo *message, TDSRequestBulkLoad request)
 		CheckPlpMessageHasEnoughBytesToRead(message, sizeof(tempLen));
 		if (offset + sizeof(tempLen) > (*message)->len)
 			return STATUS_ERROR;
-
+		Assert(sizeof(tempLen)<=sizeof(tempLen));
```

Comment:
LHS = RHS, what is the point of the assert?

Reply:
Ok will remove it.